src/js/protocols/stm32usbdfu.js

```diff
@@ -131,14 +131,13 @@ STM32DFU_protocol.prototype.closeDevice = function () {
 
     chrome.usb.closeDevice(this.handle, function closed() {
         if (checkChromeRuntimeError()) {
-            console.log('Failed to close USB device!');
+            console.log(`Failed to close USB device with Handle ID: ${self.handle.handle}`);
```

Comment:
On the mac it is very common to trigger this error, it happens when closeDevice is called as part of releaseInterface from the cleanup function that happens after a successful flash.

I don't know why Chrome nearly always reports an error at this point. It is more likely to happen when flashing F411's than F405's . It may be a timing problem.

The debugger console shows:

Chrome API Error: No such connection..